javalib/src/main/scala/java/util/LinkedHashMap.scala

```diff
@@ -0,0 +1,60 @@
+package java.util
+
+import java.{util => ju}
+
+import scala.collection.{mutable => mu}
+
+
+class LinkedHashMap[K, V] private (inner: mu.LinkedHashMap[Box[K], V],
+      initialCapacity: Int, loadFactor: Float, accessOrder: Boolean)
```

Comment:
Similarly, the primary constructor does not need `initialCapacity` and `loadFactor`.